onnxruntime/test/contrib_ops/quantize_ops_test.cc

```diff
@@ -268,7 +268,7 @@ TEST(QuantizeLinearContribOpTest, QuantizeLinear_per_tensor_half_int8) {
                                      127, -127,
                                      127, -128,
                                      127, -128});
-  test.Run();
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
```

Comment:
why run test on TensorRT EP, but not CPU/CUDA?

Reply:
It excludes TRT.

---
In reply to: [410001884](https://github.com/microsoft/onnxruntime/pull/3531#discussion_r410001884) [](ancestors = 410001884)